e2e_tests/tests/experiment/experiment.py

```diff
@@ -258,12 +258,37 @@ def is_terminal_state(state: determinedexperimentv1State) -> bool:
     )
 
 
-def trial_metrics(trial_id: int) -> Dict[str, Any]:
-    certs.cli_cert = certs.default_load(conf.make_master_url())
-    authentication.cli_auth = authentication.Authentication(conf.make_master_url(), try_reauth=True)
-    r = api.get(conf.make_master_url(), "trials/{}/metrics".format(trial_id))
-    json = r.json()  # type: Dict[str, Any]
-    return json
+class Step:
```

Comment:
maybe make this match the new proto 'metrics', https://github.com/determined-ai/determined/blob/4fb1886b8004d7bdf4d4faad7e550bbee36d06df/proto/src/determined/checkpoint/v1/checkpoint.proto#L25-L34, including optional `batch_metrics`. maybe
`def trial_metrics(trial_id: int) -> Iterable[Tuple[int, Metrics]]:`

Reply:
definitely can't be step, since steps (in this meaning of the word) have been intentionally removed from the system